src/Components/Journeys/AvailableJourneys/index.js

```diff
@@ -1,25 +1,36 @@
+import StationList from "../StationList";
+import React from 'react';
+
 const AvailableJourneys = ({journeyOptions}) => {
+
+    const secondsToHms = (time) => {
```

Comment:
Put this helper function in src/imports/library.js
It could then be unit tested.